### ISSUE TITLE
Style Projects and Fav tabs

### ISSUE THREAD

**Issue by kristinashu** [mozilla/network-pulse#887]
Based on https://github.com/mozilla/network-pulse/issues/880
Comp https://redpen.io/ns0835f2e5d9956858

**Tabs styling:** 
- same font, size and weight as the main nav except blue (same blue and buttons)
- selected and hover state has blue underline
- reduce subtitle under Projects tab ("Published Projects" and "Created Projects") to Zilla Slab Medium 16px in black

![my complete profile copy 2x](https://user-images.githubusercontent.com/1068783/33733052-44eb320a-db3d-11e7-9593-5e358977f038.png)

**Comment by kritisingh1** [comment]
@kristinashu can I take this up?

**Comment by xmatthewx** [comment]
Thanks @kritisingh1 for offering to help! This frontend work still requires some backend development, but that should be ready fairly soon. You can follow this epic https://github.com/mozilla/network-pulse/issues/860 for broader discussion. There, you'll find a link to the work on API routes.

cc @alanmoo to help facilitate

**Comment by alanmoo** [comment]
@kritisingh1 What about this ticket appeals to you? I should be able to help find something that is ready if I have a better sense of what type of work you're looking to do.

**Comment by kritisingh1** [comment]
Let alone the design part, the most appealing feature request seems to be the inclusion of the subtitles under the Projects' tab.

**Comment by mmmavis** [comment]
@kristinashu should we consider displaying some messages when no published/created/favourited entries are associated with the profile user is viewing?

**Comment by xmatthewx** [comment]
here's a few thoughts:
 - If person A is viewing person Z, I think we should just omit the tabs with no data. So maybe we lazy load it. 
 - If a profile only has one tab, we can probably just hide it altogether. 
 - If person A is viewing the own profile, it would be awesome to show favs tab with the CTA we show when mozillapulse.org/favs is empty. 

If this is all complicated, we can build and ship tabs and then follow up with this stuff. But it'll be important. Otherwise we'll send important traffic from fellowship directory to some odd looking profiles.

**Comment by mmmavis** [comment]
As an user I'd like to see the same tab options across all profiles so I know what to expect when I visit any profile page. But this is just my personal preference. I'm curious to know what @kristinashu thinks!

@xmatthewx should I pause implementation until we sort out the specs?

**Comment by xmatthewx** [comment]
Let's move forward with implementation. I can meet today to figure things out. It seems like there's plenty to build to set up tabs, pull favs, pull extended profile fields. 

For the tabs, maybe we can meet halfway for now. We can show projects and favs on all profiles, with a message in empty tabs. Very few profiles will have extended fields, so I'd like to hide that when there's no content. I'm still concerned that profiles for our highest profile people (fellows and mozfest speakers) will look deficient with empty tabs for favs and projects.

**Comment by xmatthewx** [comment]
> As an user I'd like to see the same tab options across all profiles so I know what to expect when I visit any profile page.

When I visit a github project, I don't even notice that some of them have a tab for a wiki, some for projects, some don't have either. Some repos are just more feature rich than others. But the core experience doesn't change.

**Comment by kristinashu** [comment]
Looks like we talked about this a bit in the design ticket https://github.com/mozilla/network-pulse/issues/880#issuecomment-350167327

**Comment by mmmavis** [comment]
@xmatthewx @kristinashu it would be super helpful if we could meet today and sort out the specs together and note them down either in this ticket or [the design ticket](https://github.com/mozilla/network-pulse/issues/880#issuecomment-350167327). Let's talk during stand?

**Comment by xmatthewx** [comment]
- [ ] card style remains the same
 - [ ] list of projects and headers remains the same
 - [ ] add a tab nav component
 - [ ] style tabs as listed above
 - [ ] load projects by default
 - [ ] don't load favs initially (delay if possible. wait until tab tapped if not possible)
 - [ ] follow-up: hide tabs if they're empty. hide all the tabs if only one is present
 - [ ]

**Comment by mmmavis** [comment]
partially blocked by: https://github.com/mozilla/network-pulse-api/issues/349

**Comment by xmatthewx** [comment]
@mmmavis - i think this is unblocked now. yay.

**Comment by mmmavis** [comment]
@xmatthewx there's another issue on the api side: https://github.com/mozilla/network-pulse-api/issues/358

**Comment by gideonthomas** [comment]
@mmmavis can this issue be closed now?

**Comment by mmmavis** [comment]
@gideonthomas yup yup

🎉 🎉 🎉
